template.yaml

```diff
@@ -36,18 +36,28 @@ objects:
           env:
           - name: _JAVA_OPTIONS
             value: "-Djava.net.preferIPv4Stack=true"
+          - name: JANUSGRAPH_USE_INDEXES
```

Comment:
so this will be used on each deployment, even if the first one create the indices on a PV? Will it fail if indices already exist?

Reply:
Based on work that was done by Sushmitha, this should work as expected now.